source/_integrations/nanoleaf.markdown

```diff
@@ -50,3 +51,58 @@ name:
 4. The output should include the auth token like *{"auth_token":"xxxxxxxxxxxxxxxxxxxxx"}*, copy the resulting token into your configuration
 
 If you get a 403 Forbidden message, you probably did not press the *ON* button long enough. The time-frame to get a valid token is only 30 seconds, so you have to be quick to issue the curl request.
+
+### Getting useful values for automations
+
+First set up some effects with the Nanoleaf app, and have them installed on the device.
+
+Once installed, go to "Developer Tools" and find the state of the device, eg `light.bedroom_triangles`. You should end up with something like this:
+
+```yaml
+# Example nanoleaf state:
+min_mireds: 154
+max_mireds: 833
+effect_list:
+  - Beatdrop
+  - Bedtime
+  - Blaze
+  - Cocoa Beach
+  - Cotton Candy
+  - Date Night
+  - Hip Hop
+  - Hot Sauce
+  - Jungle
+  - Lightscape
+  - Morning Sky
+  - Northern Lights
+  - Pop Rocks
+  - Prism
+  - Starlight
+  - Sundown
+  - Waterfall
+  - Vibrant Sunrise
+brightness: 155
+effect: Bedtime
+friendly_name: Triangles
+icon: 'mdi:triangle-outline'
+supported_features: 55
+```
+
+Now you can call the `light.turn_on` service with the data:
+```yaml
+# Example data to activate a nanoleaf effect:
+entity_id: light.trianglestest
+effect: Starlight
+```
```

Comment:
I feel this is out of place in this document. This is the integration documentation, not the documentation to teach how to use lights in Home Assistant.

While I get your goal, in theory, this example can be copied into hundreds of places, which is not maintainable.

Reply:
Yeah, removed this snippet.